inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
-// Copyright 2015
-// Ubiquitous Knowledge Processing (UKP) Lab and FG Language Technology
-// Technische Universität Darmstadt
-// 
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// 
-// http://www.apache.org/licenses/LICENSE-2.0
-// 
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+Getting Started
 
-= Getting started
+This Getting-Started-Guide shall assist new users to install and start
+and work with INCEpTION. It gives a quick overview (estimated time for
+reading only: approx. 15 minutes) on the main issues in order to get
+familiar with the tool and does not incude special cases and details due
+to simplicity and focus on the first steps. For further reading on any
+issue, see our main documentation.: LINK EINFÜGEN. For quick overviews,
```

Comment:
issue -> topic

Reply:
I looked it up and both works perfectly fine. But looking up, I fell in love with "key points" and chose it.